pyproject.toml

```diff
@@ -1,10 +1,23 @@
-#[project]
-#requires-python = ">=3.9"
-
 [build-system]
-requires = ["setuptools>=62.4.0"]
+requires = [
+    "Babel>=2.10.0",
+    "PyInstaller==6.12.0",
+    "pytest>=8.1",
+    "setuptools>=62.4.0",
+]
 build-backend = "setuptools.build_meta"
 
+[project]
+name = "musicbrainz-picard"
+description = "MusicBrainz Picard"
```

Comment:
In current setup.py the `summary` corresponds to `description` and is currently "The next generation MusicBrainz tagger". Not saying we shouldn't change it, but maybe set it to "The MusicBrainz audio tagger" or "MusicBrainz Picard audio tagger" or such.

Reply:
In `org.musicbrainz.Picard.appdata.xml.in` the short description is currently "MusicBrainz's music tagger". We really should settle on one being used everywhere :)